packages/cactus-plugin-persistence-ethereum/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "@hyperledger/cactus-plugin-persistence-ethereum",
+  "version": "1.1.3",
```

Comment:
@outSH `1.2.0` went out yesterday, please update this to match that version as well.

Reply:
Cheers!